cli/config/compose/services/elastic-agent/centos/docker-compose.yml

```diff
@@ -1,10 +1,10 @@
 version: '2.4'
 services:
   elastic-agent:
-    image: centos/systemd:${centos_systemdTag:-latest} 
+    image: docker.elastic.co/observability-ci/centos-systemd:latest
     container_name: ${centos_systemdContainerName}
     entrypoint: "/usr/sbin/init"
-    platform: ${stackPlatform:-linux/amd64}
+    platform: ${elasticAgentPlatform:-linux/amd64}
```

Comment:
What's the reason for splitting the platform in elasticAgent and stack? IIUC, those variables are the same

Reply:
Yeah, it seems we can reuse it, as it's not possible to use different platforms. Will merge both variables into one in a new commit on top. Good catch!

Reply:
In fact, I was thinking about removing the variable from compose files, and delegate the pull operation to Docker compose: as platform won't be set, docker compose will honour current host's platform (AMD/ARM), simplifying the configuration and code.